manim/mobject/mobject.py

```diff
@@ -2417,7 +2417,10 @@ def init_sizes(sizes, num, measures, name):
     def sort(self, point_to_num_func=lambda p: p[0], submob_func=None):
         """Sorts the list of :attr:`submobjects` by a function defined by ``submob_func``."""
         if submob_func is None:
-            submob_func = lambda m: point_to_num_func(m.get_center())
```

Comment:
**(question):** Is there a particular reason this was changed? If not, it should probably not be changed in this PR.

Reply:
I didn't change any code. I was only working with docstrings. I might have made a mistake while updating my local branch. @hydrobeam Please advise on how I go about fixing this. Thank you!

Reply:
I think precommit-ci did it

Reply:
Could you just manually undo the changes? That should fix it